README.md

```diff
@@ -642,6 +642,7 @@ Waste collection schedules in the following formats and countries are supported.
 - [Movar IKS](/doc/source/movar_no.md) / movar.no
 - [Oslo Kommune](/doc/source/oslokommune_no.md) / oslo.kommune.no
 - [ReMidt Orkland muni](/doc/source/remidt_no.md) / remidt.no
+- [Stavanger Kommune](/doc/source/sandnes_no.md) / sandnes.kommune.no
```

Comment:
[Sandnes Kommune](/doc/source/sandnes_no.md) / sandnes.kommune.no

Reply:
Fixes itself when changing the `TITLE` in the py file (and running update_doculinks.py)